src/components/MyAccount/RequestsTab/FreezeButton.tsx

```diff
@@ -38,6 +39,12 @@ const FreezeButton = ({ hold, patron }: { hold: Hold; patron: Patron }) => {
     },
   }
 
+  useEffect(() => {
+    if (isDisabled === false) {
```

Comment:
This only really works when the initial state for `isDisabled` is `null`. It's then set to either true or false but the initial `null` value helps _not_ focus on the button when the page loads.

Reply:
Could this focus invocation happen after line 69?

Reply:
I tried it. It didn't work. I think it's because when the function is declared, the `buttonRef` is still null. So unfortunately it has to be in an `useEffect`.